app/assets/stylesheets/styles/components/pages/_how_it_works.css.scss

```diff
@@ -1,18 +1,20 @@
+// @import "compass/css3/flexbox";
+
 .how-it-works {
   background-color: #dcccb3;
-  height: 70%;
+  height: $hero-height-small;
   max-width: initial;
   overflow: hidden;
   position: absolute;
   // Initially position the how-it-works section above the screen
-  top: -200%;
+  top: -100%;
```

Comment:
I thought -200% was overzealous and unnecessary, but I guess there was a reason >_<
Not sure why that's happening… perhaps the `100%` refers to that container's height and not the parent's?